cpp/oneapi/dal/data/graph_service_functions.hpp

```diff
@@ -36,46 +30,46 @@ namespace oneapi::dal::preview {
 
 /// Get number of vertexes in a graph.
 ///
-/// @tparam G  Type of graph
+/// @tparam Graph  Type of graph
 ///
-/// @param [in]   g  graph
+/// @param [in]   graph  Input graph
 ///
-/// @return Result is vertex_size_type of G.
-template <typename G>
-constexpr auto get_vertex_count(const G &g) noexcept -> vertex_size_type<G> {
-    return get_vertex_count_impl(g);
+/// @return Result is vertex_size_type of Graph.
```

Comment:
sounds a bit strange - return type of Graph
a routine returns  'a value' of some type, this function returns vertex count for the graph

Reply:
Documentation will be updated in separate pull request.